recipes/nomkl/run_test.py

```diff
@@ -0,0 +1,7 @@
+import subprocess
+import json
+
+args = ("conda", "list", "mkl", "--json")
+
+mkl_list = json.loads(subprocess.check_output(args))
+assert "mkl" not in mkl_list
```

Comment:
I don't think there is a chance that we would install MKL in the test env. Should we add some dep that possibly could use it?

Reply:
> I don't think there is a chance that we would install MKL in the test env.

Zero chances. Just added to satisfy the linter.

> Should we add some dep that possibly could use it?

I though about that but how to we catch a failed env creation to pass a test?

Reply:
Ohhh I meant a dep that would use openblas or mkl. So like, we install numpy and make sure mkl doesn’t appear

Reply:
That makes sense. Let's try it.

Reply:
BTW, just tried to build this locally forcing `mkl` here and got the conflict as expected!